selfdrive/controls/lib/events.py

```diff
@@ -945,4 +945,10 @@ def joystick_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.SubMaster,
     ET.NO_ENTRY: NoEntryAlert("LKAS Disabled"),
   },
 
+  EventName.vehicleSensorsInvalid: {
+    ET.PERMANENT: soft_disable_alert("LKAS Unavailable: Vehicle Sensors Calibrating"),
+    ET.IMMEDIATE_DISABLE: ImmediateDisableAlert("LKAS Unavailable: Vehicle Sensors Invalid"),
+    ET.NO_ENTRY: NoEntryAlert("LKAS Unavailable: Vehicle Sensors Calibrating"),
+  },
```

Comment:
I want to let the user know there is an eventual end to the alert, but `Calibrating` might not make sense for all the cases we'll use this alert for.

Reply:
let's make it a good alert for the current case. we can think about other cases when they pop up

Reply:
I think it's okay as is now then

Reply:
look good?